src/linuxspi.c

```diff
@@ -403,10 +403,17 @@ static int linuxspi_parseextparams(const PROGRAMMER *pgm, const LISTID extparms)
   for (ln = lfirst(extparms); ln; ln = lnext(ln)) {
     extended_param = ldata(ln);
 
-    if (strcmp(extended_param, "disable_no_cs") == 0) {
+    if (str_starts(extended_param, "disable_no_cs")) {
```

Comment:
Why str_starts() here and not str_eq()?

Reply:
Woops, that's an error! Sorry about that, I'll have it fixed.